### PR TITLE
Buff jani belt

### PR DESCRIPTION
**Pull request by Guidesu**
Kinda simple.
Buffed Jani belts.
They hold now:
Galoshes
Bucket

### CODE REVIEW

**Comment by BurpleBineapple** [comment]
Trash bag/Bluespace Trash bag I would remove. Those two can hold massive objects like toolboxes, etc. Having that fit in a belt is a little gamebreaking.

**Comment by Guidesu** [comment]
No no no.
The belt cant hold any trashbag if they have a item inside of it

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -307,8 +307,13 @@
 		/obj/item/clothing/gloves,
 		/obj/item/device/assembly/mousetrap,
 		/obj/item/weapon/crowbar/prybar,
-		/obj/item/clothing/mask/plunger
-		)
+		/obj/item/clothing/mask/plunger,
+		/obj/item/weapon/storage/bag/trash,
+		/obj/item/weapon/storage/bag/trash/bluespace,
```
Comment (io-util) [inline code/game/objects/items/weapons/storage/belt.dm]: As Purple already commented, this isn't the best idea.

**Review by io-util — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (io-util) [reply code/game/objects/items/weapons/storage/belt.dm]: Remove them and it should be fine.

**Comment by io-util** [comment]
> No no no.
> The belt cant hold any trashbag if they have a item inside of it

They technically can.

**Comment by Guidesu** [comment]
Righ right okay that makes sense

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (Guidesu) [reply code/game/objects/items/weapons/storage/belt.dm]: fixed

**Review by io-util — APPROVED** [review]
Looks fine from my side.

**Review by BurpleBineapple — APPROVED** [review]
Also seems fine

**Pull request merged**